lib/js/jsconsole.nim

```diff
@@ -0,0 +1,32 @@
+#
+#
+#            Nim's Runtime Library
+#        (c) Copyright 2012 Andreas Rumpf
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+
+## Wrapper for the `console` object for the `JavaScript backend
+## <backends.html#the-javascript-target>`_.
+
+when not defined(js) and not defined(Nimdoc):
+  {.error: "This module only works on the JavaScript platform".}
+
+type Console* {.importc.} = ref object of RootObj
+
+{.push importcpp .}
+
+proc log*[A](console: Console, a: A)
```

Comment:
I don't know how. The various arguments can be of different types, hence the only thing that would work is `varags[untyped]`. But that works only in the context of a macro, and eventually we need to write an actual binding to the underlying functionality, which is a method.

I don't have a quick solution, but I tought it would be better to have the functionality - even if limited - than not. Even more so considering that this is a debug feature.

A quick improvement would be to publish various versions for various arities, say up to 4 arguments. Bu this gets hairy if we also want to support logging strings nicely - now the overlading works because the version for strings is not generic